gatsby/src/reports/reports.js

```diff
@@ -0,0 +1,248 @@
+              filter: {
+                frontmatter: { reviewed: { lt: "2020-01-01" } }
+                fields: { slug: { regex: "/^((?!changelog).)*$/" } }
+              }
+            ) {
+              edges {
+                node {
+                  id
+                  frontmatter {
+                    title
+                    reviewed
+                  }
+                  fields {
+                    slug
+                  }
+                }
+              }
+            }
+            unreviewedDocs: allMdx(
+              filter: {
+                frontmatter: { reviewed: { eq: null }, title: { ne: "" } }
+                fields: { slug: { regex: "/^((?!changelog).)*$/" } }
+              }
+            ) {
+              edges {
+                node {
+                  id
+                  frontmatter {
+                    title
+                    reviewed
+                  }
+                  fields {
+                    slug
+                  }
+                }
+              }
+            }
+          }
+        `}
+        render={data => {
+          const [search, setSearch] = useState("")
+          const reviewedPages = data.allReviewedDocs.edges
+          const reviewedTertiaryPages = data.allReviewedDocs.edges.filter(
+            page => {
+              return page.node.fields.slug.match(/\/guides(\/[a-z,\-]*){2}/)
+            }
+          )
+          //console.log("Tertiary Pages: ", tertiaryPages) // For debugging
+          const oldPages = data.staleDocs.edges
+          const unreviewedPages = data.unreviewedDocs.edges
+          //console.log("Unreviewed Docs Array: ", unreviewedPages) // For debugging
+          const unreviewedTertiaryPages = data.unreviewedDocs.edges.filter(
+            page => {
+              return page.node.fields.slug.match(/\/guides(\/[a-z,\-]*){2}/)
+            }
+          )
+          console.log("Unreviewed Tertiary Pages: ", unreviewedTertiaryPages) // For debugging
+          return (
+            <Layout>
+              <h1>Reports</h1>
+
+              <div className="form-group">
+                <div className="input-group">
+                  <div className="input-group-addon">
+                    <i className="fa fa-search" />
+                  </div>
+                  <input
+                    type="text"
+                    id="command-search"
+                    className="form-control"
+                    placeholder="Filter by Title"
+                    onChange={e => setSearch(e.target.value)}
+                    value={search}
+                  />
+                  <div
+                    style={{ background: "#fff; cursor:pointer" }}
+                    className="input-group-addon"
+                    id="clear-filter"
+                    onClick={e => setSearch("")}
+                  >
+                    <span className="fa fa-times" />
+                  </div>
+                </div>
+              </div>
```

Comment:
This section was shamelessly stolen from `/terminus/commands`, and creates a form for users to enter a page name and filter the results.